frontend/lib/src/theme/getColors.ts

```diff
@@ -93,13 +93,6 @@ export const createEmotionColors = (genericColors: {
     borderColor: derivedColors.fadedText10,
     borderColorLight: derivedColors.fadedText05,
 
-    docStringModuleText: genericColors.bodyText,
-    docStringTypeText: genericColors.green70,
-    docStringContainerBackground: transparentize(
-      genericColors.secondaryBg,
-      0.6
-    ),
```

Comment:
Removing some of these theme color variables to keep our them mostly based on semantic colors that we want to make configurable, and not once purley adapted from other colors.